napari/_qt/qt_viewer.py

```diff
@@ -216,7 +217,7 @@ def _update_camera(self):
                 self.view.camera.viewbox_key_event = viewbox_key_event
                 self.viewer.reset_view()
 
-    def screenshot(self):
+    def screenshot(self, path=None):
```

Comment:
Can you add a doc string for `path` here (same one you used in `Viewer.py` is fine.